src/core/lib/security/authorization/stdout_logger.cc

```diff
@@ -0,0 +1,75 @@
+// Copyright 2023 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <grpc/support/port_platform.h>
+
+#include "src/core/lib/security/authorization/stdout_logger.h"
+
+#include <cstdio>
+#include <initializer_list>
+#include <memory>
+#include <string>
+
+#include "absl/status/status.h"
+#include "absl/status/statusor.h"
+#include "absl/strings/str_format.h"
+#include "absl/strings/string_view.h"
+#include "absl/time/clock.h"
+#include "absl/time/time.h"
+
+#include <grpc/grpc_audit_logging.h>
+#include <grpc/support/log.h>
+
+#include "src/core/lib/json/json.h"
+
+namespace grpc_core {
+namespace experimental {
+
+constexpr absl::string_view kName = "stdout_logger";
+const char kLogFormat[] =
+    "{\"grpc_audit_log\":{\"timestamp\":%d,\"rpc_method\":\"%s\",\"principal\":"
+    "\"%s\",\"policy_name\":\"%s\",\"matched_rule\":\"%s\",\"authorized\":%s}}";
```

Comment:
Will this have implicit newlines, and if so do we want a multi-line log?

Reply:
Good catch! I meant to add a newline to flush the buffer but forgot. Added.

Reply:
I actually meant the other way around, do we want _any_ newlines here at all, or one big single line that is the audit log?

Reply:
As you also saw in the Go PR, we want one log entry to be one-liner.